docs/tutorials/ort-format-model/packaging-and-deploy.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Packaging and deploy
+parent: Deploy ORT format model for mobile device and web
```

Comment:
Is this page going to have content before checkin? If not should we skip adding it until we have the content?

Reply:
yes i will update content before checkin